src/InteractiveWindow/Editor/History.cs

```diff
@@ -159,11 +164,16 @@ private Entry Get(string pattern, int step)
 
         internal Entry GetNext(string pattern = null)
         {
+            //if you are at the last history entry or history navigation pointer is
+            //not initialized then return null as there is no next entry
+            if ((_current == Length - 1) || (_current == -1)) return null;
```

Comment:
@amcasey I thought move method was written with a goal to search entire history so I did not change it. We are certainly not using it to do so ATM but we may want to in future.